databricks/koalas/namespace.py

```diff
@@ -273,7 +273,7 @@ def read_delta(path: str, version: Optional[str] = None, timestamp: Optional[str
     Examples
     --------
     >>> ks.range(1).to_delta('%s/read_delta/foo' % path)
-    >>> ks.read_delta('%s/read_delta/foo' % path)
+    >>> ks.read_delta('%s/read_delta/foo' % path)  # doctest: +SKIP
```

Comment:
Those tests are related with https://github.com/databricks/koalas/pull/639/files#r313350742.

I will make a separate PR to fix it.